src/OpenApi/src/Services/OpenApiComponentService.cs

```diff
@@ -42,7 +42,26 @@ private readonly ConcurrentDictionary<(Type, ParameterInfo?), JsonObject> _schem
     {
         OnSchemaGenerated = (context, schema) =>
         {
-            schema.ApplyPrimitiveTypesAndFormats(context.TypeInfo.Type);
+            var type = context.TypeInfo.Type;
+            // Fix up schemas generated for IFormFile, IFormFileCollection, Stream, and PipeReader
```

Comment:
Add any tests with Stream and PipeReader?

Reply:
Add tests here. It appears that Stream/PipeReader get the same treatment as FormFiles when procued by the ApiExplorer layer for MVC. Added a note about this in https://github.com/dotnet/aspnetcore/issues/55349.